packages/admin/cms-admin/src/dam/CurrentDamFolderContext.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+
+type FolderContextType = {
+    folderId: string | undefined;
+};
+
+const FolderContext = React.createContext<FolderContextType>({ folderId: undefined });
```

Comment:
Please rename to CurrentDamFolderContext. The type and variable can have the same name